From 6a3d5e6ff6f605b93b0d585231fcea7cbae3f3fb Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 24 Aug 2021 22:36:46 +0300
Subject: [PATCH] [SW-36150] gaudi2: pll: allow any target compile

Preboot needs to initialize the SRAM PLLs for asic to be able to serve
BMC requests in time.
Allow the PLLs code to be compiled for any target.

Change-Id: I8ffc704945725c5ef5aa472531eb6ed30cfc586a
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/pll.c b/arch/gaudi2/common/pll.c
index 792820c..4940a61 100644
--- a/arch/gaudi2/common/pll.c
+++ b/arch/gaudi2/common/pll.c
@@ -50,8 +50,6 @@
 	return out_freq;
 }
 
-#ifdef TARGET_IS_PLDM
-
 #define GAUDI2_PLL_CFG		0x100
 #define GAUDI2_PLL_CFG_EN	BIT(0)
 #define GAUDI2_PLL_CFG_LOCK	BIT(16)
@@ -118,5 +116,3 @@
 {
 	regmap_writel(1, cfg->base + GAUDI2_PLL_DIV_SEL_DELAY);
 }
-
-#endif /* TARGET_IS_PLDM */
